Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/NodePaletteWidget.cpp

```diff
@@ -88,6 +89,9 @@ namespace EMStudio
         m_palette->SetupNodePalette(config);
         m_palette->hide();
         m_layout->addWidget(m_palette);
+        // TODO(#13382): Investigate why we need to apply the style here even though it's applied globally
```

Comment:
nit: I know this is super minor but could you please update the 'TODO' to instead say `GHI-13382` instead please

```
// GHI-13382 - Investigate why we need to apply the style here even though it's applied globally
```

Reply:
Sure.

Reply:
Do we want to retain the `TODO` so it shows up in tools or just the `// GHI-13382 ...`?

Reply:
Just `GHI-13382`

Reply:
Done